src/tile.cpp

```diff
@@ -271,7 +268,7 @@ const Creature* Tile::getBottomVisibleCreature(const Creature* creature) const
 		} else {
 			for (auto it = creatures->rbegin(), end = creatures->rend(); it != end; ++it) {
 				if (!(*it)->isInvisible()) {
-					const Player* player = (*it)->getPlayer();
+					auto player = (*it)->getPlayer();
```

Comment:
Could this be `const auto`
```suggestion
					const auto player = (*it)->getPlayer();
```